prov-api/routes/model_run/model_run.py

```diff
@@ -272,19 +272,31 @@ async def update_model_run(
             detail=f"Failed to validate an entity's ID in the record, error: {error_message}."
         )
 
-    res = await submit_model_run_update_job(
-        username=roles.user.username,
-        payload=ProvLodgeUpdatePayload(
-            model_run_record_id=payload.model_run_id,
-            updated_record=payload.record,
-            reason=payload.reason,
-            revalidate=True,
-            # encrypted user info
-            user_info=user_cipher
-        ),
-        config=config)
-
-    return PostUpdateModelRunResponse(session_id=res)
```

Comment:
Yeah - would prefer you don't change this. In the integration test code just check the status is 200 - no need to change this so you can look at status field.